composer.json

```diff
@@ -30,9 +30,12 @@
             "mito\\sentry\\tests\\": "tests/"
         }
     },
-    "extra": {
-        "asset-installer-paths": {
-            "bower-asset-library": "vendor/bower"
+    "config": {
+        "fxp-asset": {
```

Comment:
Only the root package's config is used, so this is only used when working on this package.